src/lib/rest/restReply.cpp

```diff
@@ -174,6 +179,16 @@ void restReply(ConnectionInfo* ciP, const std::string& _answer)
         MHD_add_response_header(response, "Access-Control-Allow-Origin", restAllowedOrigin);
       }
     }
+
+    if (ciP->verb == OPTIONS)
+    {
+      MHD_add_response_header(response, "Access-Control-Allow-Headers", "Content-Type, User-Agent, Fiware-Service, Fiware-Servicepath, Ngsiv2-AttrsFormat, Fiware-Correlator, X-Forwarded-For, X-Real-IP, X-Auth-Token");
```

Comment:
Content-Type and User-Agent are not needed to be declared here. I think

Reply:
In order to have better extensibility or maintainability of the code it would be great to define the headers as an array of constants, so that when new headers are needed it would be enough to only touch the constant array.

Reply:
> Content-Type and User-Agent are not needed to be declared here
* Content-Type can be out of the scope of simple headers depending on its value. Please see Step 10 of [6.2 Preflight Request](https://www.w3.org/TR/cors/#simple-header)
* Agreed with User-Agent I will remove it. 

> In order to have better extensibility or maintainability of the code it would be great to define the headers as an array of constants, so that when new headers are needed it would be enough to only touch the constant array.

Yes, this is planned for the next PR.

Reply:
Fixed in f97a6f1